solutions/swb-reference/integration-tests/tests/multiStep/costCenter.test.ts

```diff
@@ -16,39 +16,51 @@ describe('multiStep costCenter test', () => {
   test('create, get, update, list, delete', async () => {
     console.log('Creating Cost Centers');
     const accountId = setup.getSettings().get('defaultHostingAccountId');
-    const { data: createdCostCenterA } = await adminSession.resources.costCenters.create({
+    const createCostCenterAResponse = await itAdminSession.resources.costCenters.create({
       accountId,
       name: 'costCenterA'
     });
+    expect(createCostCenterAResponse.status).toEqual(201);
 
-    const { data: createdCostCenterB } = await adminSession.resources.costCenters.create({
+    const createCostCenterBResponse = await itAdminSession.resources.costCenters.create({
       accountId,
       name: 'costCenterB'
     });
+    expect(createCostCenterBResponse.status).toEqual(201);
+
+    const costCenterA = createCostCenterAResponse.data;
+    const costCenterB = createCostCenterBResponse.data;
 
     console.log('Get Cost Center A');
-    const { data: getCostCenterA } = await adminSession.resources.costCenters
-      .costCenter(createdCostCenterA.id)
-      .get();
-    expect(getCostCenterA).toMatchObject(createdCostCenterA);
+    const responseCostCenterA = await itAdminSession.resources.costCenters.costCenter(costCenterA.id).get();
+    expect(responseCostCenterA.status).toEqual(200);
```

Comment:
nit: `expect(responseCostCenterA).toEqual(expect.objectContaining({status: 200, data:costCenterA}))` should work here.

Reply:
I think we probably want to avoid compound `expect`s when there are only two things being checked because the error reporting is easier to read, especially if you have responses with large amounts of text in them.